apps/proposer/src/proposer.mjs

```diff
@@ -1,9 +1,43 @@
 /* eslint-disable import/no-unresolved */
+/* eslint-disable no-await-in-loop */
 /**
 Module that runs up as a proposer
 */
 import logger from '@polygon-nightfall/common-files/utils/logger.mjs';
 
+const TIMER_CACP = process.env.TIMER_CACP || 30;
```

Comment:
Add these two variables to config

Reply:
added, thanks

Reply:
if you are going to change @imagobea, then also use config vars instead of env vars (that's why we defined them).

The idea to have vars in config is that one can see which things are configurable. If configuration is only from env vars, is difficult to get a full picture of all the parameters needed.